pkg/cfg/config.go

```diff
@@ -504,20 +504,68 @@ func (c *config) mergeStruct(prefix string, settings interface{}, options ...Mer
 	return c.mergeMsi(prefix, msi, options...)
 }
 
-func (c *config) readEnvironment(prefix string, input *mapx.MapX) *mapx.MapX {
+func (c *config) readEnvironmentFromStructKeys(prefix string, structKeys []mapx.StructKey) *mapx.MapX {
+	environment := mapx.NewMapX()
+
+	var key, envKey string
```

Comment:
Why are these outside of the loop? They seem to be used only at some places and are always assigned before they are used

Reply:
true, not needed anymore after the latest changes